### DISCUSSION TITLE
Use "vehicle_data" even in "suspended"/"offline" teslamate state

### DISCUSSION THREAD

**Discussion by GaPhi** [teslamate-org/teslamate#2463] [Ideas]
A quick look on the tesla app logs shows a new query parameter : "let_sleep=true" for the "vehicle_data" command.

I think this would be a good idea to use it, isn't it ?

I am not sure what it exactly means:
- Do not wake up the car?
- Do not prevent it to fall asleep?
- Force it to fall asleep immediately?

I have updated teslajs and node-red-contrib-tesla libraries to test this...

**Comment by Acelya-9028** [comment]
@GaPhi Hi, have you any news about this?
